src/components/organisms/EarthEditor/CanvasArea/index.tsx

```diff
@@ -61,6 +62,7 @@ const CanvasArea: React.FC<Props> = ({ className, isBuilt }) => {
       engine="cesium"
       isEditable={!isBuilt}
       isBuilt={!!isBuilt}
+      isInEditor={isInEditor !== false}
```

Comment:
```suggestion
      isInEditor={!!isInEditor}
```

Reply:
Here i want to set the value to `true` except got `isInEditor` as `false`; `CanvasArea` is a child component of `EarthEditor` so it should always be `true` for `isInEditor`, but i found the implementation of preview page as it use the `CanvasArea` directly therefore i pass the params `false` from preview page. So that's why i did not use `!!isInEditor`

Reply:
Updated.